server/src/streamAPI.ts

```diff
@@ -95,8 +95,9 @@ export const createStreamAdminToken: RequestHandler = async (req, res) => {
     }
     console.log("> Creating stream admin token for room " + roomId);
     try {
-        // exp is optional (by default the token is valid for an hour)
-        const exp = Math.round(new Date().getTime() / 1000) + 60 * 60;
+        // exp is optional (by default the token is valid for an hour).
+        // set exp to be 6 hours.
```

Comment:
We are doing a 24 hour live stream soon. Wondering if this should be 36h then?
Or would we be okay with the token expiring mid stream? What happens if the token expires?

Reply:
ah yeah, i'll bump the expiration -- I forget exactly what happens but I ran into some sort of issue before.